lib/streamlit/runtime/caching/cached_message_replay.py

```diff
@@ -247,7 +248,7 @@ def calling_cached_function(
         nested_call = False
         ctx = get_script_run_ctx()
         if ctx:
```

Comment:
nit: can we remove one level of indentation here now that we shouldn't have to check for the existence of a `ctx` to run this code?